_posts/2019-12-13-cockpit-209.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Cockpit 209
+author: mvo
+date: 2019-12-13
+tags: cockpit linux
+slug: cockpit-209
+category: release
+summary: Cockpit with Overview improvements
+comments: 'true'
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/). We release regularly.  Here are the release notes from version 209.
+
+### A new design for the Overview page
+
+The landing page has been completely redesigned:
```

Comment:
Let's expand this a bit -- emphasize health notifications, group information into categories, graphs move to subpage, better discoverability of the hw info subpage.

Reply:
Done.